_overviews/scala3-book/methods-most.md

```diff
@@ -279,16 +288,15 @@ val d = new Dog
 d.speak()   // prints "Woof"
 ```
 
-Methods can also be marked as `private`.
-This makes them private to the current class, so they can’t be called nor overridden in subclasses:
+Methods can also be marked as `private`. This makes them private to the current class, so they can’t be called nor overridden in subclasses:
```

Comment:
Making it one line doesn't change rendering, but "nor" should be "or".

Reply:
"Or" done! "This" is not a question of rendering, but "This" implies some idea development in the scope of a paragraph. Anyway...